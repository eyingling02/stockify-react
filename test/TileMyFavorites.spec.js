import React from 'react'
import TileMyFavorites from '../js/TileMyFavorites.jsx'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

test('MyFavorites snapshot test', () => {
  const component = shallow(<TileMyFavorites />)
  const tree = shallowToJson(component)
  expect(tree).toMatchSnapshot()
})
