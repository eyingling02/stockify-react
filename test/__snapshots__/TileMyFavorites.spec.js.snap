exports[`test MyFavorites snapshot test 1`] = `
<div
  className="outer">
  <div
    className="buy">
    <a
      href="javascript:;">
      BUY
    </a>
  </div>
  <div
    className="inner">
    <a
      href="livingroom.html"
      target="_blank">
      <h2 />
      <p>
        Today G/L: 
        %
      </p>
      <p>
        Total G/L: 
        %
      </p>
      <p>
        Total Picks: 
         days
      </p>
    </a>
  </div>
  <div
    className="sell">
    <a
      href="javascript:;">
      SELL
    </a>
  </div>
</div>
`;
